### DISCUSSION TITLE
Deploy changes bug

### DISCUSSION THREAD

**Discussion by VPoinsot** [planetscale/discussion#284] [Bug Reports] [answered]
Hello all :)
I've been trying to deploy a database since yesterday, but nothing.
![image](https://user-images.githubusercontent.com/9005683/190878925-c394afae-a0ae-4e5d-83dc-1285db14adb0.png)

Have any of you had this bug?

Thanks

**Comment by fooforge** [comment, marked as answer]
@VPoinsot Dang, looks like you have hit a bug. Sorry about that! We'll look into it today and I'll report back once it has been fixed.

**Reply by fooforge** [reply]
@VPoinsot We have resolved the issue, you should be good to go again now.

**Reply by VPoinsot** [reply]
thanks :)
